crates/libcontainer/src/syscall/linux.rs

```diff
@@ -507,6 +507,13 @@ impl Syscall for LinuxSyscall {
                     caps::drop(None, CapSet::Bounding, *c)?
                 }
             }
+            CapSet::Ambient => {
+                value.iter().for_each(|c| {
```

Comment:
Please keep the original comment as is.
It would also be nice to add a comment mentioning that if it fails, we emit a warn log and continue with the subsequent processing.